controller/tests/controller_server/csi_controller_server_test.py

```diff
@@ -422,11 +449,18 @@ def test_create_volume_idempotent_no_source_succeeds(self, storage_agent):
     def test_create_volume_with_wrong_secrets(self, a_enter):
         self._test_create_object_with_wrong_secrets(a_enter)
 
+    @patch("controller.controller_server.csi_controller_server.get_agent")
+    def test_create_volume_no_pool(self, storage_agent):
+        self._prepare_create_volume_mocks(storage_agent)
+        self.request.parameters = {"pools_by_system": str({"u1": pool, "u2": "other_pool"})}

```

Comment:
is the `pools_by_system` name purposely invalid?

Reply:
it's a leftover, I removed all of them